.github/workflows/tests_unmocked.yml

```diff
@@ -17,9 +17,9 @@ jobs:
         path: ${{ env.pythonLocation }}
         key: cache_v1_${{ env.pythonLocation }}-${{ hashFiles('requirements.txt') }}
     - name: Install Dependencies and lightly
-      run: pip install . 
+      run: pip install .
     - name: run unmocked tests
       run: |
         export LIGHTLY_SERVER_LOCATION=${{ secrets.LIGHTLY_SERVER_LOCATION }}
-        export TOKEN=${{ secrets.DUMMY_USER_TOKEN_STAGING }}
+        TOKEN=${{ secrets.DUMMY_USER_TOKEN_STAGING }}
         bash tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh $TOKEN
```

Comment:
why not use the secret directly in the command as opposed to put it in a variable first?

Reply:
I changed that and it ran through: https://github.com/lightly-ai/lightly/runs/4352405806?check_suite_focus=true